po/cs.po

```diff
@@ -0,0 +1,644 @@
+msgid "Count lines, symbols and words"
+msgstr "Počítání řádků, symbolů a slov"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:16
+msgid "Format JSON and XML"
+msgstr "Formátování JSON a XML"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:17
+msgid "Escape and unescape strings"
+msgstr "Únikové a neúnikové řetězce"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:18
+msgid "Convert JSON to YAML and vice versa"
+msgstr "Převádění JSON na XML a naopak"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:19
+msgid "Filter lines"
+msgstr "Filtrování řádků"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:20
+msgid "Replace substrings and regular expressions"
+msgstr "Nahrazení podřetezců a regulárních výrazů"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:22
+msgid "...and so on."
+msgstr "...a tak dále."
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:27
+msgid "Gleb Smirnov"
+msgstr "Gleb Smirnov"
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:69
+msgid "Minor UI changes"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:70
+msgid "Add XML formatter"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:75
+msgid "Some bugfixes and improvements"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:78
+msgid "Bring back tools with arguments"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:83
+msgid "Redesign the application"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:84
+msgid "Add custom tools support"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:85
+msgid "Improve search"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:86
+msgid "Add ability to text load from file and save to file"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:93
+msgid "Exit by Ctrl+Q"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:94
+msgid "Monospace argument entry"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:95
+msgid "Update new runtime"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:102
+msgid "Sort tools"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:103
+msgid "Filter tools"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:104
+msgid "Minify C-like code"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:111
+msgid "Add more tools"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:112
+msgid "Make tools popover more powerful"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:113
+msgid "Redesign icon"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:114
+msgid "General improvements"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:119
+msgid "Add tools for replacement and removing"
+msgstr ""
```

Comment:
You just skipped some strings. If I remember correctly, this will use empty string instead translation.

Reply:
I resolved this and problem below.